src/styles/card.scss

```diff
@@ -0,0 +1,148 @@
+.main {
+  display: flex;
+  justify-content: center;
+  padding-top: 50px;
+}
+
+.main__container {
+  width: 944px;
+  height: 856px;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.main__block1 {
+  margin-bottom: 50px;
+  width: 944px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.main__block2 {
+  width: 944px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.card__text {
+  margin-top: 40px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  width: 170px;
+  line-height: 18px;
+  font-size: 12px;
+  color: #060b35;
+}
+
+.card__product-Code {
+  margin-top: 4px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__container-feedback {
+  display: flex;
+  align-items: flex-end;
+}
+
+.stars {
+  display: flex;
+}
+
+.card__stars {
+  margin-top: 16px;
+}
+
+.stars__star {
```

Comment:
stars is another BEM-block, so you need to create a new file for it

Reply:
done